x-pack/plugins/security_solution/public/flyout/left/context.tsx

```diff
@@ -29,16 +29,24 @@ export interface LeftPanelContext {
    */
   indexName: string;
   /**
-   * Retrieves searchHit values for the provided field
+   * Maintain backwards compatibility // TODO remove when possible
    */
-  getFieldsData: (field: string) => unknown | unknown[];
+  scopeId: string;
+  /**
+   * An object containing fields by type
+   */
+  browserFields: BrowserFields | null;
   /**
    * An array of field objects with category and value
    */
   dataFormattedForFieldBrowser: TimelineEventsDetailsItem[] | null;
+  /**
+   * Retrieves searchHit values for the provided field
+   */
+  getFieldsData: (field: string) => unknown | unknown[];
```

Comment:
instead of casting to unknown, consider reuse `GetFieldsData` type in `x-pack/plugins/security_solution/public/common/hooks/use_get_fields_data.ts`

Reply:
yes thank you, not sure why I didn't use it. I remember you commenting on this on a previous PR also :D